src/plugins/discover/public/application/angular/context/query_parameters/actions.ts

```diff
@@ -46,7 +58,7 @@ export function getQueryParameterActions(filterManager, indexPatterns) {
     filterManager.addFilters(newFilters);
     if (indexPatterns) {
       const indexPattern = await indexPatterns.get(indexPatternId);
-      await popularizeField(indexPattern, field.name, indexPatterns);
+      await popularizeField(indexPattern, (field as IFieldType).name, indexPatterns);
```

Comment:
We are casting here something that (theoretically) could be a `string` according to our types. Why are we sure here it's not a `string` anymore? Otherwise I think we should have a proper typeguard in place (btw it could have been this code was broken beforehand, so ideally trust your types more than the existing code :D)